src/status_im/communities/core.cljs

```diff
@@ -780,3 +780,24 @@
                                 :community-id community-id
                                 :public-key   public-key
                                 :role-id      role-id})]})
+
+(rf/defn mute-chat-failed
+  {:events [:community/mute-community-failed]}
+  [{:keys [db]} community-id muted? error]
+  (log/error "mute community failed" community-id error)
+  {:db (assoc-in db [:communities community-id :muted] (not muted?))})
+
+(rf/defn mute-chat-toggled-successfully
+  {:events [:community/mute-community-successful]}
+  [_ community-id]
+  (log/debug "muted chat successfully" community-id))
```

Comment:
afaik, It would be better to update the `db` in here, as the network request has successfully gone through here 👍 
although perhaps @cammellos or someone else can confirm/ shed some light on this.